jsonnet-mode.el

```diff
@@ -242,14 +248,19 @@ If not inside of a multiline string, return nil."
 (defun jsonnet-eval-buffer ()
   "Run jsonnet with the path of the current file."
   (interactive)
-  (let ((buffer-to-eval (buffer-file-name)))
+  (let ((file-to-eval (buffer-file-name))
+        (search-dirs jsonnet-lib-dirs))
```

Comment:
Why this instead of just referring to jsonnet-library-search-directories on line 259?

Reply:
Once we're inside of the `with-current-buffer` form, we're no longer reading a buffer-local variable (here, `jsonnet-lib-dirs`) for the Jsonnet buffer we're evaluating; instead, we'd be reading the variable bound within the "*jsonnet-output*" buffer, which is the wrong one. It took me quite a while to realize why that wasn't working.

My first move was to use `buffer-local-value` to fetch the value from the original buffer, but I thought it was harder to read, as it required saving a reference to the original buffer in order to reference a variable from it later.